Makefile

```diff
@@ -9,14 +9,19 @@ test: generate
 	go list ./... | xargs go test
 
 linux:
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build \
+	apt-get install -y libpcre3-dev
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=1 go build \
 			 -ldflags "-X main.version=$(BUILD_VERSION)" \
 			 -o dist/falco-linux-amd64 ./cmd/falco
 
-darwin:
-	GOOS=darwin GOARCH=amd64 go build \
+darwin_amd64:
+	brew install pcre
```

Comment:
`brew install pcre` adds a significant overhead at every compilation because brews tries to update. is there a way we can avoid that?